client/services/code_preprocessors/PythonCodePreprocessorServiceSpec.js

```
@@ -69,6 +81,15 @@ describe('PythonCodePreprocessorService', function() {
           [[true, true], [false, false], [true, false]])
       ).toEqual("[[True, True], [False, False], [True, False]]");
     });
+
+    it('should throw error for unsupported inputs', function() {
```

should throw an error if the input type is unsupported